### PR TITLE
RS+RMD/YJ/RULE19-18

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
Question)

In the RDS, ```AHJ_RA_compare``` comes out of nowhere. Could you elaborate on in which case ```AHJ_RA_compare``` is True/False?

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section19/section19rule18.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                    total_fan_power += zone_fan_elec_power
+
+            fan_power_W_CFM = total_fan_power / supply_cfm_b
+
+            return {"fan_power_W_CFM": fan_power_W_CFM}
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            fan_power_W_CFM = calc_vals["fan_power_W_CFM"]
+
+            return std_equal(FAN_POWER_LIMIT, fan_power_W_CFM) or (
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule18.py:130]: Good point! Fixed.

File: rct229/rulesets/ashrae9012019/section19/section19rule18.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_fan_object_electric_power import (
+    get_fan_object_electric_power,
+)
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule18.py:24]: I'm a bit confused. The existing code already used the ```baseline_system_type_compare``` function and RDS already mentions to use the function. Am I missing something now?

File: rct229/rulesets/ashrae9012019/section19/section19rule18.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            id="19-18",
+            description="or baseline system 1 and 2, the total fan electrical power (Pfan) for supply, return, exhaust, and relief shall be = CFMs × 0.3, where, CFMs = the baseline system maximum design supply fan airflow rate, cfm.",
+            ruleset_section_title="HVAC - General",
+            standard_section=" Section G3.1.2.9",
+            is_primary_rule=True,
+            rmr_context="ruleset_model_instances/0",
+            list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    def is_applicable(self, context, data=None):
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule18.py:50]: Please let me know if further updates are needed.

File: rct229/rulesets/ashrae9012019/section19/section19rule18.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (weilixu) [reply rct229/rulesets/ashrae9012019/section19/section19rule18.py:24]: You are correct. My mistake.

**Review by weilixu — APPROVED** [review]

**Pull request merged**